orttraining/orttraining/core/graph/pipeline_transformer.cc

```diff
@@ -68,12 +71,49 @@ NodeArg& CreateNodeArg(Graph& graph, const NodeArg* base_arg) {
   return graph.GetOrCreateNodeArg(new_name, &type_proto);
 }
 
-Status AddRecordBackward(Graph& graph,
-                       Node* send_bw,
-                       std::vector<std::string>& new_input_names,
-                       std::vector<std::string>& new_output_names,
-                       std::string &event_id_tensor_name,
-                       std::string &output_tensor_name) {
+// Return mirror variables for node_args. The i-th output element mirrors node_args[i]
```

Comment:
>he i-th output element mirro [](start = 43, length = 28)

The description and function name look generic. Could we just call it event node args for clarity?

Reply:
It's name is general because its function is actually general. I can rename this helper function to `CreateMirrorNodeArgs`. What do you think?

---
In reply to: [425969251](https://github.com/microsoft/onnxruntime/pull/3823#discussion_r425969251) [](ancestors = 425969251)